docs/getting_started.rst

```diff
@@ -28,17 +28,29 @@ Run the following commands to install the dependencies and use CMake to build th
     util/scripts/install-deps-ubuntu.sh
     mkdir build
     cd build
-    cmake ../src
+    cmake ../src -DCMAKE_INSTALL_PREFIX=<open3d_install_directory>
     make -j
 
-Install Python binding module by running the following script.
+The ``CMAKE_INSTALL_PREFIX`` argument is optional.
+
+.. tip:: In order to build the Open3D Python module with a particular Python version in mind set the either the ``PYBIND11_PYTHON_VERSION`` or the ``PYTHON_EXECUTABLE`` cmake option.
+
+.. code-block:: bash
+
+    cmake ../src -DPYBIND11_PYTHON_VERSION=2.7
+    # or
+    cmake ../src -DPYTHON_EXECUTABLE=/usr/bin/python
```

Comment:
The python issue is addressed in docs/compilation.rst. How about providing a link?

Reply:
done